workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt

```diff
@@ -98,7 +98,13 @@ class WorkflowViewStub @JvmOverloads constructor(
    * The id to be assigned to new views created by [update]. If the inflated id is
    * [View.NO_ID] (its default value), new views keep their original ids.
    */
-  @IdRes var inflatedId: Int
+  @IdRes var inflatedId: Int = NO_ID
+    set(value) {
+      require(value == NO_ID || value != id) {
+        "inflatedId and id must be distinct"
```

Comment:
Nit: include the names of the id resources in this (and the other) message?

Reply:
Oh, of course. Will follow up.

Reply:
https://github.com/square/workflow-kotlin/pull/184